scrapers/sexuallybroken.yml

```diff
@@ -0,0 +1,27 @@
+name: sexuallybroken
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.sexuallybroken.com/sb
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="is-size-5 has-text-weight-bold"]/text()
+      Date:
+        selector: //span[@class="tag is-dark"]/text()
+      Details: 
+        selector: //div[@class="is-size-6 has-text-white-ter"][3]/text()
+        concat: "<br>"
+        postProcess:
+          - replace:
+            - regex: <br>
+              with: "\n"
+      Performers:
+        Name: //span[@class="tag is-dark"]/a/text()
```

Comment:
the span doesnt't work, probably something changed
this works for the scenes i tested
```yaml
        Name: //div[@class="is-size-6 has-text-white-ter"]/a[@class="tag is-dark"]/text()
```